### PR TITLE
fix continue button partially hidden on the Change password screen

### LINKED ISSUE #20399: "Confirm password" and "Continue" buttons are partially hidden on the "Change password" screens
# Bug Report

## Problem 

Screenshot 2024-06-07 at 14 45 12

Screenshot 2024-06-07 at 14 44 58

#### Expected behavior

https://www.figma.com/design/JlpPhJp0SMnEyBQy4nOZHo/Settings-for-Mobile?node-id=6638-398995&t=aeuFuf81AWWmvmTY-4

Screenshot 2024-06-07 at 14 51 28

Screenshot 2024-06-07 at 14 51 43

### Reproduction

1. Go to profile
2. **>** `Password`
3. **>** `Change password`

### Additional Information

- Status version: nightly 07/06
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/20399

### Screenshots
<details>
<summary>Images</summary>
<table border="0">
 <tr>
    <td><b style="font-size:30px">Android</b></td>
    <td><b style="font-size:30px">IOS</b></td>
 </tr>
 <tr>
    <td>

Screenshot

</td>
    <td>

Screenshot

</td>

 </tr>
</table>
</details>

status: ready

### CODE REVIEW

File: src/status_im/contexts/profile/settings/screens/password/change_password/style.cljs [hunk 1]
```diff
@@ -48,7 +48,7 @@
    :justify-content   :space-between})
 
 (def bottom-part
-  {:margin-bottom   (- (safe-area/get-bottom) 12)
+  {:margin-bottom   (+ (safe-area/get-bottom) 12)
```
Comment (Parveshdhull) [inline src/status_im/contexts/profile/settings/screens/password/change_password/style.cljs]: only related change (probably was typo)

Comment (Parveshdhull) [reply src/status_im/contexts/profile/settings/screens/password/change_password/style.cljs]: wait

Comment (Parveshdhull) [reply src/status_im/contexts/profile/settings/screens/password/change_password/style.cljs]: It was not typo, still not correct.

File: src/status_im/contexts/profile/settings/screens/password/change_password/style.cljs [hunk 2]
```diff
@@ -48,7 +49,7 @@
    :justify-content   :space-between})
 
 (def bottom-part
-  {:margin-bottom   (- (safe-area/get-bottom) 12)
+  {:margin-bottom   (if platform/ios? (safe-area/get-bottom) 12)
```
Comment (Parveshdhull) [inline src/status_im/contexts/profile/settings/screens/password/change_password/style.cljs:52]: related change

File: src/legacy/status_im/ui/components/keyboard_avoid_presentation.cljs [hunk 3]
```diff
@@ -10,7 +10,7 @@
     (reagent/as-element
      (into [react/keyboard-avoiding-view
             (update props
-                    :keyboardVerticalOffset
+                    :keyboard-vertical-offset
```
Comment (Parveshdhull) [inline src/legacy/status_im/ui/components/keyboard_avoid_presentation.cljs:13]: Only rename from camel to kebab case

**Comment by seanstrom** [comment]
@Parveshdhull Hey thanks for the fix 🙌  
Can you include some screenshots for these changes (for both iOS and Android) inside the PR description please 🙏

**Comment by Parveshdhull** [comment]
> Can you include some screenshots for these changes (for both iOS and Android) inside the PR description please 🙏

@seanstrom Updated description with screenshots

**Review by seanstrom — APPROVED** [review]
Looks good! ✅

**Comment by pavloburykh** [comment]
@Parveshdhull thanks for the PR. Failed e2e are not PR related. Ready for merge.

**Pull request merged**